Assets/Scripts/Obstacle.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class Obstacle : MonoBehaviour //, IObstacles
```

Comment:
nit: If interface `IObstacles` is not in use, I suggest removing it to reduce the confusion.